src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -206,17 +206,33 @@
           {{ $t('Search Listing.Label.Subtitles') }}
         </div>
       </div>
-      <ft-icon-button
-        class="optionsButton"
-        :icon="['fas', 'ellipsis-v']"
-        :title="$t('Video.More Options')"
-        theme="base-no-default"
-        :size="16"
-        :use-shadow="false"
-        dropdown-position-x="left"
-        :dropdown-options="dropdownOptions"
-        @click="handleOptionsClick"
-      />
+      <div class="buttonStack">
+        <font-awesome-icon
+          v-if="showDeArrowToggle || deArrowTogglePinned"
+          :title="deArrowToggleTitle"
+          :icon="['far', 'dot-circle']"
+          class="optionsButton deArrowToggleButton"
+          :class="{ alwaysVisible: deArrowTogglePinned }"
+          :style="{
+            padding: 10 + 'px',
+            fontSize: 16 + 'px'
+          }"
```

Comment:
As these values are static/don't change, please move them to the `deArrowToggleButton` class in the SCSS file.